bpf/init.sh

```diff
@@ -568,7 +568,7 @@ if [ "$HOSTLB" = "true" ]; then
 
 	CALLS_MAP="cilium_calls_lb"
 	COPTS="-DLB_L3 -DLB_L4"
-	if [ "$IP6_HOST" != "<nil>" ]; then
+	if [ "$IP6_HOST" != "<nil>" ] || [ "$IP4_HOST" != "<nil>" ]; then
```

Comment:
Is there any chance this will cause the datapath load to just break if someone compiles out IPv6?

Reply:
I've tested it explicitly with `--enable-ipv4=true --enable-ipv6=false` which was the whole point to have the v4-in-v6 being handled even when v6 if off since the packet will end up as v4 on the wire. Do you have anything specific in mind wrt breakage?

Reply:
Oh, I was thinking about the underlying kernel. That said I'm not sure whether there are any well-known kernels that completely disable IPv6.

Reply:
Ah, okay, will check it out and get back.

Reply:
Indeed needs fixing, thanks!